packages/js/product-editor/src/blocks/notice/block.json

```diff
@@ -0,0 +1,39 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/block.json",
+	"apiVersion": 2,
+	"name": "woocommerce/product-has-variations-notice",
+	"title": "Notice",
+	"category": "woocommerce",
+	"description": "Notice description",
+	"keywords": [ "products", "notice" ],
+	"textdomain": "default",
+	"attributes": {
+		"id": {
+			"type": "string"
+		},
+		"title": {
+			"type": "string"
+		},
+		"content": {
+			"type": "string"
+		},
+		"buttonText": {
+			"type": "string"
+		},
+		"type": {
+			"type": "string"
+		}
+	},
+	"supports": {
+		"align": false,
+		"html": false,
+		"multiple": true,
+		"reusable": false,
+		"inserter": false,
+		"lock": false,
+		"__experimentalToolbar": false
+	},
+	"usesContext": [ "selectedTab" ],
```

Comment:
Could remove this now as well, given you are not making use of the `selectedTab` context anymore